src/main/java/com/minecolonies/items/ModItems.java

```diff
@@ -9,6 +9,11 @@
     public static final Item scanTool    = new ItemScanTool();
     public static final Item caliper     = new ItemCaliper();
 
+    public static final Item itemAchievementProxySettlement = new ItemAchievementProxy("sizeSettlement");
+    public static final Item itemAchievementProxyTown       = new ItemAchievementProxy("sizeTown");
+    public static final Item itemAchievementProxyCity       = new ItemAchievementProxy("sizeCity");
+    public static final Item itemAchievementProxyMetropolis = new ItemAchievementProxy("sizeMetropolis");
+
     // deactivated for now
     // public static final Item compost    = new ItemCompost();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)